index.html

```diff
@@ -0,0 +1,225 @@
+      <div class="top_head">
+        <ul class="top_head_ul">
+          <li class="top_head_items_icon">
+            <a href="#">
+              <img src="images/icons/icons_facebook_f.svg" alt="facebook">
+            </a>
+          </li>
+          <li class="top_head_items_icon">
+            <a href="#">
+              <img src="images/icons/icons_twitter.svg" alt="twitter">
+            </a>
+          </li>
+          <li class="top_head_items">
+            <a href="#">English</a>
+          </li>
+          <li class="top_head_items">
+            <a href="#">My Page</a>
+          </li>
+          <li class="top_head_items">
+            <a href="#">Logout</a>
+          </li>
+        </ul>
+      </div>
+    <!-- top header end  -->
+    <!-- Navbar section start -->
+      <nav class="nav_desktop">
+        <div class="navbar_container">
+          <a href="#home" id="brand">
+            <img class="top_header_logo" src="images/logo/Logo.png" alt="logo">
+          </a>
+          <ul class="desktop desktop-nav">
+            <li class="navbar-links">
+              <a href="index.html">Home</a>
+            </li>
+            <li class="navbar-links">
+              <a href="#program_section">Programs</a>
+            </li>
+            <li class="navbar-links">
+              <a href="#speakers_section">Speakers</a>
+            </li>
+            <li class="navbar-links">
+              <a href="#partners">Sponsor</a>
+            </li>
+            <li class="navbar-links">
+              <a href="about.html">About</a>
+            </li>
+              <li class="navbar-links campaign_link">
+                <a href="#contact"> CC Campaign</a>
+              </li>
+          </ul>
+          <div class="mobile-menu-bars mobile" id="menu-bars">
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
+          </div>
+        </div>
+      </nav>
+    <!-- navbar section end  -->
+    <!-- navbar for mobile start  -->
+      <nav class="overlay overlay-slide-left" id="overlay">
+        <ul class="mobile-nav-items">
+        <li id="nav-1"><a href="#home" class="nav-mobile-links">Home</a></li>
+        <li id="nav-1"><a href="#program_section" class="nav-mobile-links">Programs</a></li>
+        <li id="nav-2"><a href="#speakers_section" class="nav-mobile-links">Speakers</a></li>
+        <li id="nav-3"><a href="#partners" class="nav-mobile-links">Sponsor</a></li>
+        <li id="nav-3"><a href="about.html" class="nav-mobile-links">About</a></li>
+        <li id="nav-3"><a href="#" class="nav-mobile-links">Campaign</a></li>
+        </ul>
+      </nav>
+      <!-- navbar for mobile end  -->
+    </header>
+      <!-- main header start -->
+    <section class="header_part" id="home">
+
+      <div class="header_part_main_div">
+        <h2 class="hello-text"> "Hello! Sharing World"</h2>
+
+        <div class="header_part_text_div">
```

Comment:
- [ ] Could you please check the `**font-size** `and `**padding** `of your main title, particularly in the mobile versions? It appears to **be smaller than what's shown in the template.** It would be awesome if you could make some adjustments to ensure it matches the intended design.🙏

   ### Your design:
   ![image](https://github.com/Nelofarzabi/Capstone_Project/assets/107952724/3477f09a-83eb-45c9-8a7b-852ca0b699b2)

   ### Behance design:
   ![image](https://github.com/Nelofarzabi/Capstone_Project/assets/107952724/a7b5ece4-37f3-4034-bbc6-46a9dbdbe1d3)